easylistdutch/block_third_party_whitelist.txt

```diff
@@ -1,72 +1,26 @@
-@@||adhese.be/tag/tagv1.js$domain=nieuwsblad.be
-@@||ads.spilgames.com/ad/pb/1/*/*?$script,domain=spel.nl|spelletjes.nl
-@@||adserver.adtech.de/addyn/3.0/$script,domain=spel.nl|spelletjes.nl
-@@||adserver.adtech.de/addyn|3.0^$script,domain=9292.nl
-@@||adserver.adtech.de/multiad/$script,domain=spel.nl|spelletjes.nl
-@@||aka-cdn-ns.adtech.de/apps/*/wallpaper.jpg$domain=9292.nl
 @@||aka.spotxcdn.com/integration/ados/v1/ados.js$script,domain=www.radioluisteren.fm
-@@||arval.solution.weborama.fr/fcgi-bin/comptage_wreport.fcgi?$image,domain=arval.nl
-@@||blackbird.zoomin.tv/adhandler/$domain=standaard.be
-@@||cdn.adnxs.com/blank.js?$domain=9292.nl
-@@||cdn.adnxs.com/megatag.js$domain=9292.nl
-@@||cdn.eyewonder.com/*/video/$object,domain=radioveronica.nl
 @@||cdn.spotxcdn.com/integration/easi/v1/easi.js$script,domain=www.radioluisteren.fm
-@@||cdnservices.com/webads_smarttags.v2.js$domain=vi.nl
-@@||doubleclick.net^*;tile=$script,domain=syndication.vmma.be|vtm.be
-@@||doubleclick.net^*adj/hbvl/homepage;tile=15;sz=160x600;$script,domain=hbvl.be
-@@||easyads.eu/api/Advertisements/$domain=marktfeed.nl
-@@||easyads.eu/api/v2/AdvertentiePlanet/me/advertisements/$domain=advertentieplanet.nl
-@@||g.doubleclick.net/gampad/ads?$script,domain=marktplaats.nl
-@@||g.doubleclick.net/gampad/ads?$xmlhttprequest,domain=radioluisteren.fm
-@@||google.com/ads/search/module/ads/*/search.js$domain=zoover.be|zoover.nl
-@@||google.com/adsense/$subdocument,domain=sedo.nl
-@@||google.com/adsense/search/ads.js$domain=zoover.be|zoover.nl
-@@||googleads.g.doubleclick.net/pagead/ads?ad_type=video_flash_text_image&$object,domain=kinderspelletjes.nl
-@@||googletagservices.com/tag/js/gpt.js$domain=telegraaf.nl
-@@||ib.adnxs.com/ab?$script,domain=9292.nl
-@@||ib.adnxs.com/mtj?$script,domain=9292.nl
-@@||imageceu1.247realmedia.com/0/Player/Player_NL_Home/342x204_mensuel_NLnew.jpg
+@@||faktor.io^$3p,domain=consent.talpanetwork.com
 @@||images.marktnet.eu/adimg/
-@@||imasdk.googleapis.com/flash/core/*/adsapi.swf$object,domain=538.nl
+@@||imasdk.googleapis.com^$3p,domain=storage.y8.com
 @@||imasdk.googleapis.com/js/core/bridge*.html$subdocument,domain=nos.nl
-@@||imasdk.googleapis.com/js/core/bridge3.*.html#goog_$subdocument,domain=2be.be|vtm.be|vtmkzoom.be
-@@||imasdk.googleapis.com/js/sdkloader/ima3.js$domain=2be.be|vtmkzoom.be
+@@||imasdk.googleapis.com/js/core/bridge3.*.html#goog_$subdocument,domain=vtm.be
+@@||imasdk.googleapis.com/js/sdkloader/ima3.js$script,domain=www.hbvl.be
+@@||imasdk.googleapis.com/js/sdkloader/ima3.js$domain=vtm.be
 @@||js.spotx.tv/ados/v1/106185.js$script,domain=www.radioluisteren.fm
-@@||maps.gstatic.com/maps-api-*/adsense.js$domain=opentot.nl
-@@||media.adrcdn.com/ads/Mini/*/vast.xml$xmlhttprequest,domain=radio10.nl
-@@||media.ztat.net/media/teaser/$image,domain=zalando.nl
-@@||nl.gmads.net/crossdomain.xml$object,domain=radioveronica.nl
-@@||nl.gmads.net/r?gmtr=$object,domain=radioveronica.nl
+@@||nl.y8.com/games/*$domain=nl.y8.com
```

Comment:
This is a crazy filter. It is way to broad pus it belongs to EasyList not EasyList Dutch

Reply:
This is a exception on EasyList for a dutch domain and thus belong to EasyList Dutch. 
And this rule is necessary (and thus broad), because otherwise their are to many games that don't function normal.

Reply:
> This is a exception on EasyList for a dutch domain and thus belong to EasyList Dutch.
> And this rule is necessary (and thus broad), because otherwise their are to many games that don't function normal.

@@||y8.com/games/*.js?referer=$xhr,domain=y8.com
@@||imasdk.googleapis.com/js/$script,domain=y8.com

Reply:
But understand that this isn't dutch specific. This rules must go to the general EasyList